### ISSUE TITLE
Function Comment sniff request?

### ISSUE THREAD

**Issue by bkdotcom** [squizlabs/PHP_CodeSniffer#909]
my "IDE" (Sublime w/ DocBlockr)  generates doc block comments.  
It will generate something like this

```
/**
 * [myfunction description]
 *
 * @param [type] $varname [description]
 *
 * @return [type] [description]
 */
function myfunction($varname) {
}
```

It's up to me to "fill in the blanks"

Neither the Generic or Squiz Commenting sniffs object to me leaving the placeholders (I'll _assume_ the PEAR sniffs don't as well)

I _believe_ these are common placeholders generated by many IDEs -n- plugins...   Not sure if Generic or some new test should have a built in check for them.

**Comment by aik099** [comment]
Yes, sniffs just check that text is in correct format. The `[` and `]` are not forbidden in DocBlock and therefore they aren't reported as an error.

For example Zend Studio generates DocBlocks (see http://files.zend.com/help/Zend-Studio-10/zend-studio.htm#phpdoc_comments.htm) that look like this:

```php
/**
 * Enter description here...
 *
 * @param unknown_type $a
 * @param unknown_type $b
 * @return unknown
 */
function add ($a, $b) {
    return $a + $b;
}
```

As a human I can understand that this DocBlock needs to be populated with correct data, but there are no `[` in there to check for.

**Comment by bkdotcom** [comment]
Perhaps forbid "Enter description here...",   "[type]", "unknown_type", "unknown", etc?

I suppose "unknown_type" and "unknown" could be classnames..

This could be a solution looking for a problem, but I know I tend to procrastinate and figure "I'll fill that out later"

**Comment by gsherwood** [comment]
Closing as I've had no more interest or PRs for this.
